core/designsystem/src/main/java/com/google/samples/apps/nowinandroid/core/designsystem/component/Button.kt

```diff
@@ -71,7 +71,12 @@ fun NiaFilledButton(
         colors = colors,
         contentPadding = contentPadding,
         content = {
-            ProvideTextStyle(value = MaterialTheme.typography.labelSmall) {
+            ProvideTextStyle(
+                value = if (small)
```

Comment:
@ricknout Is this alligned with our design specs?

Reply:
Yes